PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1746,7 +1772,23 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 			{
 				enableFindDlgItem(IDREDOTMATCHNL, false);
 			}
+
 			enableProjectCheckmarks();
+
+			if (inSelAutoChangeSoAbortSearch)
+			{
+				// show this to the user only after all UI changes have been made,
+				// so user doesn't see something out-of-date if he looks at the
+				// search window while the msgbox is still open
+				NppParameters& nppParamInst = NppParameters::getInstance();
+				(nppParamInst.getNativeLangSpeaker())->messageBox(
+					"FindAutoChangeOfInSelectionWarning",
+					_hSelf,
+					L"The \"In selection\" checkbox state has been automatically modified. Please verify the search condition before performing the action.",

```

Comment:
Please break down the one line sentence `The "In selection" checkbox state has been automatically modified. Please verify the search condition before performing the action.` into 2 lines.